packages/admin/resources/views/components/header/heading.blade.php

```diff
@@ -1,3 +1,3 @@
-<h1 {{ $attributes->class(['text-2xl font-bold tracking-tight md:text-3xl filament-header-heading']) }}>
+<h1 {{ $attributes->class(['text-2xl font-bold tracking-tight filament-header-heading']) }}>
```

Comment:
Unrelated to this PR, but I'll update it for consistency.

```suggestion
<h1 {{ $attributes->class(['filament-header-heading text-2xl font-bold tracking-tight']) }}>
```